### PR TITLE
Logo Updates

### PR DESCRIPTION
**Pull request by sachasmart-weavik**
### Description

> Logo Update. Used TinyPng to reduce the filesizes. Probably should wait for @luansievers dark mode to view in storybook. 
### Checklist

- [x] `CHANGELOG` has been updated (_if appropriate_)
- [x] Environment variables updated (_if appropriate_)
fa-logo

### CODE REVIEW

**Review by v1n33th — APPROVED** [review]
Awesome...
One minor suggestion is to place the image files in the [`public`](https://nextjs.org/docs/basic-features/static-file-serving) folder as it's the recommended way to serve static files. But maybe there could be a reason to place it under the design system. Thoughts?

**Comment by sachasmart-weavik** [comment]
My approach was to replace the logos with the FA logos in the locations where Goldfinch had them. Some of them were in the public folder, where I kept the same names as the Goldfinch team. With the design system, I'm not sure why Goldfinch has it like that - maybe for ease of use...?

**Comment by v1n33th** [comment]
I'm not sure either, why they did it like this. I think we can leave it as it is :)

File: packages/client3/components/design-system/logo/logo.stories.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 export default {
   title: "Components/Logo",
-  component: GoldfinchLogo,
+  component: FreeArtistsLogo,
 } as Meta;
 
-export const LogoStory: Story<typeof GoldfinchLogo> = () => {
-  return <GoldfinchLogo className="h-10 w-10" />;
+export const LogoStory: Story<typeof FreeArtistsLogo> = () => {
+  return <FreeArtistsLogo className="h-10 w-10" />;
```
Comment (luansievers) [inline packages/client3/components/design-system/logo/logo.stories.tsx:11]: Can you maybe add class as an arg, so we can change on the story props?

File: packages/client3/components/design-system/logo/logo.stories.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import { Story, Meta } from "@storybook/react";
 
-import { GoldfinchLogo } from ".";
+import { FreeArtistsLogo } from ".";
 
 export default {
   title: "Components/Logo",
```
Comment (luansievers) [inline packages/client3/components/design-system/logo/logo.stories.tsx:6]:
```suggestion
  title: "FAD/Components/Logo",
```

**Review by luansievers — CHANGES_REQUESTED** [review]
`packages/client3/public/android-chrome-192x192.png`
`packages/client3/public/android-chrome-512x512.png`
`packages/client3/public/apple-touch-icon.png`
`packages/client3/public/mstile-150x150.png`

Needs to be the same size as before with the new logo.

Another thing, why did you move the `setup` stuff? Ideally, we leave things outside of client3 untouched in order to make it easier to merge stuff from the goldfinch repo (unless is stuff we created)

Another thing, should we abbreviate things to `FA` or `FAD`? @v1n33th @sachasmart-weavik , and I ask this just because of the images uploaded in this pr are `fa`, we should chose one.

File: packages/client3/components/design-system/logo/logo.stories.tsx (hunk 2 above)
Comment (sachasmart-weavik) [reply packages/client3/components/design-system/logo/logo.stories.tsx]: > 

1. Got new logos from design to meet the dimensions. 

2. Fixed the setup directory. Apologizes looks like those two scripts were copied into the git folder

3. Changed fa-[logo] to FAD-logo as per group decision.

**Review by luansievers — APPROVED** [review]

**Pull request merged**